clients/iOS/Grapevine/Grapevine/Event.swift

```diff
@@ -18,11 +18,9 @@ class Event: NSObject, Mappable {
     var eventId: Int!
     var feedId: Int!
     var timeProcessed: NSDate!
-    var startTimeNS: NSDate!
-    var startTime: CVDate!
-    var endTimeNS: NSDate!
-    var endTime: CVDate!
-    var repeatsWeekly: Int!
+    var startTime: EventTime!
```

Comment:
It honestly depends. Generally, you only implicitly unwrap them when you know they'll always have a value. So for me, when I build events off of the api response, there are certain values that i know I will always have and in that case it is more convenient to unwrap them right away instead of waiting until you need that data and then having to check whether it exists before you access it.